README.md

```diff
@@ -1,17 +1,68 @@
 # FastQC-nf bioinformatics workflow
 
+ - [Description](#description)
+ - [User Guide](#user-guide)
+ - [Additional notes](#additional-notes)
+ - [Acknowledgements](#acknowledgements)
+
 ## Description
-FastQC-nf is a pipeline for quality control in both illumina short reads and Pacbio long reads. It's written in nextflow. 
 
+<p align="center">
+:wrench: This pipeline is currently under development :wrench:
+</p>
+
+FastQC-nf is a Nextflow workflow for evaluating the quality of high-throughput sequencing reads. It employs [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/) to assess the quality of individual fastq files. FastQC metrics are then aggregated into a single HTML report using [MultiQC](https://multiqc.info/). This pipeline accepts both Illumina short read and PacBio long read datasets. 
+
+## Diagram 
+![diagram](workflow1-1.bmp)
 
 ## User guide
 
-## Component tools
+### Set up 
+
+Under development.
+
+#### Install Nextflow and Singularity
+
+To run this pipeline you must have Nextflow (>=20.07.1) and Singularity installed on your machine. All tools are run using containers. 
+
+* [Nextflow installation instructions](https://www.nextflow.io/docs/latest/getstarted.html)
+* [Singularity installation instructions](https://docs.sylabs.io/guides/3.0/user-guide/installation.html)
```

Comment:
Added instructions for environment set up to ensure users/testers have dependencies installed before trying to run the workflow.